### PR TITLE
fix hard coded breaks ipv6

### PR DESCRIPTION
**Pull request by JieJhih**
Signed-off-by: JieJhih <jiejhihjhang@gmail.com>
Fix: #2289

### CODE REVIEW

**Comment by thaJeztah** [comment]
ping @arkodg @selansen @joeabbey PTAL

**Comment by selansen** [comment]
CI failure doesnt seem related 
--- FAIL: TestNetworkDBCRUDMediumCluster (12.32s)
	networkdb_test.go:112: Network existence verification failed
	networkdb_test.go:112: Network existence verification failed

File: ipvs/netlink.go [hunk 1]
```diff
@@ -424,6 +424,9 @@ func assembleDestination(attrs []syscall.NetlinkRouteAttr) (*Destination, error)
 		switch attrType {
 		case ipvsDestAttrAddress:
 			ip, err := parseIP(attr.Value, syscall.AF_INET)
+			if len(attr.Value) > 4 && err != nil {
```
Comment (selansen) [inline ipvs/netlink.go]: Can you change it like below. Always better to check err first and then go with second validation. 
if err != nil  && len(attr.Value) >  {

**Comment by kwanhur** [comment]
it seems no `d.AddressFamily` return by ip_vs
https://git.kernel.org/pub/scm/utils/kernel/ipvsadm/ipvsadm.git/tree/libipvs/ip_vs_nl_policy.c#n31

**Comment by thaJeztah** [comment]
the ipvs package was moved to a separate repository; https://github.com/moby/ipvs. If you still think this change is relevant, could you open a new PR in that repository?

**Pull request closed without merging**